compiler/src/parsing/lexer.mll

```diff
@@ -197,13 +202,19 @@ rule token = parse
   | "*" { STAR }
   | "/" { SLASH }
   | "%" { PERCENT }
+  | "+=" { PLUSEQ }
```

Comment:
‼️ 

How did we lex this stuff before? Or was it part of the parser?

Reply:
Just a part of the parser! Having it lexed as a single token makes the error handlers much nicer.